pkg/build/pipelines/autoconf/configure.yaml

```diff
@@ -16,6 +16,11 @@ inputs:
       The GNU triplet which describes the build system.
     default: ${{host.triplet.gnu}}
 
+  opts:
+    description: |
+      Options to pass to the ./configure command.
+    default: ''
+
```

Comment:
nice , was this intended to be part of the same PR

Reply:
Yeah the e2e tests fail without this.

Reply:
(If you expand the rest of the file, you can see we are using `${{inputs.opts}}` even though it's not defined.)